installer/prepare.ps1

```diff
@@ -21,7 +21,7 @@ $msiDependencies = @(
     } ;
     @{
         'Name'='nodejs';
-        'Url'="https://nodejs.org/dist/v4.4.7/node-v4.4.7-x86.msi";
+        'Url'="https://nodejs.org/dist/v12.19.0/node-v12.19.0-x86.msi";
```

Comment:
did you test the connector with node v12?

Reply:
yes, it works on node 12. However I still need to test out the installer, but I can't do so without jenkins building it.